packages/discovery-provider/plugins/pedalboard/apps/solana-relay/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.23",
   "private": true,
   "scripts": {
-    "start": "ts-node ./src/index.ts",
-    "dev": "ts-node-dev ./src/index.ts",
+    "start": "npx tsx ./src/index.ts",
+    "dev": "npx tsx watch ./src/index.ts",
```

Comment:
is this faster than ts-node? is this susceptible to the same npx pinning issue we saw on CI awhile ago?

Reply:
Speed isn't why I switched it - ts-node didn't support `cause` from es2022

And good point, should try and pin this